### DISCUSSION TITLE
How to use with operagx as driver?

### DISCUSSION THREAD

**Discussion by wmoural** [ultrafunkamsterdam/undetected-chromedriver#2036] [Q&A]
I believe that using uc with operagx as driver would make scrapping much faster. But i don't know how. Can someone help me?

**Comment by nathanfallet** [comment]
It is not supported since this library is made using cdp (chrome devtool protocol) which is a protocol specific to Chrome/Chromium browsers.

**Comment by ultrafunkamsterdam** [comment]
You could try nodriver, successor of uc. Which is about the fastest you can get

**Reply by nathanfallet** [reply]
Does Opera support CDP?
